src/main/java/pl/Lukasz/Calories/Controller/MealsController.java

```diff
@@ -0,0 +1,66 @@
+package pl.Lukasz.Calories.Controller;
+
+import org.springframework.beans.factory.annotation.*;
+import org.springframework.data.repository.query.Param;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.Lukasz.Calories.Entity.Meals;
+import pl.Lukasz.Calories.Entity.Users;
+import pl.Lukasz.Calories.Service.MealsService;
+import pl.Lukasz.Calories.Service.UsersService;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@RestController
+public class MealsController {
+    @Autowired
+    private MealsService service;
+
+    @GetMapping("/meals")
+    public List<Meals> list(@Param("user") Integer user) {
+        return service.listAll(user);
+    }
+
+    @GetMapping("/meals/calories")
+    public Integer sumCaloriesToday(){
+        return service.sumCaloriesToday();
+    }
+
+    @GetMapping("/meals/{id}")
+    public ResponseEntity<Meals> get(@PathVariable Integer id) {
+        try {
+            Meals meals = service.get(id);
+            return new ResponseEntity<Meals>(meals, HttpStatus.OK);
+        } catch (NoSuchElementException e) {
```

Comment:
Some Runtime Exception should never be caught - such as NullPointer or NoSuchElement - it's better to add if statements. Handling such exceptions is a bad code smell.